src/components/timeline/Line.tsx

```diff
@@ -25,15 +25,20 @@ const Line = React.memo((props: LinePropsInternal) => {
 
   const renderStartPoint = () => {
     if (entry) {
-      return <View style={[styles.entryPoint, {backgroundColor: color}]}/>;
+      return (
+        <View
+          style={[styles.entryPoint, {backgroundColor: color}]}
+          testID={`${testID}.${top ? 'startPoint' : 'endPoint'}`}
```

Comment:
Why do we need the testID here? What are we testing?

Reply:
This could be useful when testing the line and the end/start point colors.